### PR TITLE
✨ Render amp-script hash to Pixi template

### LINKED ISSUE #4813: Pixi: amp-script hash missing
https://amp.dev/vi/page-experience/
image
image

cc @matthiasrohmer could you have a look since you setup the amp-script src?

### PR DESCRIPTION
**Pull request by matthiasrohmer**
Sorry, had a misconception that the AMP optimizer will add the hash, but it only does so for inline-scripts.

Fixes #4813, probably related to https://github.com/ampproject/amp.dev/issues/4818#issuecomment-705389563.

### CODE REVIEW

File: pixi/src/ui/page-experience.hbs [hunk 1]
```diff
@@ -2,3 +2,4 @@
 {{#htmlWebpackPlugin.files.js }}
 {% set src = '{{ . }}' %}
 {{/htmlWebpackPlugin.files.js }}
+{% set cspHash = '{{ cspHash }}' %}
```
Comment (patrickkettner) [inline pixi/src/ui/page-experience.hbs:5]: would  there ever a case for multiple scripts on the page?

**Review by patrickkettner — APPROVED** [review]

**Comment by patrickkettner** [comment]
LGTM!

File: pixi/src/ui/page-experience.hbs (hunk 1 above)
Comment (matthiasrohmer) [reply pixi/src/ui/page-experience.hbs:5]: In this context yes, only one hash, as Pixi's webpack config will only ever emit one bundle.

Comment (matthiasrohmer) [reply pixi/src/ui/page-experience.hbs:5]: ... and yes, rendering `src` in a loop then is confusing 😅

**Comment by patrickkettner** [comment]
fwiw, it doesn't appear to be the issue in #4818

**Pull request merged**